docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -2,7 +2,7 @@
 = Adding `container` components to a devfile
 
 [role="_abstract"]
-Use the `container` component type to define an image-based configuration of a container in a workspace. This allows to incorporate custom tools into the workspace. A devfile can contain one or more components of the `container` type. The `image` attribute identifies the component.
+Use the `container` component type to define an image-based configuration of a container in a workspace. This `container` component type allows you to incorporate custom tools into the workspace. A devfile can contain one or more components of the `container` type. The `component name` attribute identifies each component, including the container.
```

Comment:
```suggestion
To incorporate custom tools into the workspace, you can define an image-based configuration of a container in a workspace. You can do this by using the `container` component type. 

A devfile can contain one or more components of the `container` type. The `component name` attribute identifies each component, including the container.
```

Reply:
I've restructured this abstract by describing the user story and high-level user task first. Then, I mention the feature or capability. 

I moved the details about the feature to a separate paragraph.